test/entity_test.rb

```diff
@@ -63,12 +63,6 @@ class Camera
         book.instance_variable_get(:@published).must_equal false
       end
 
-      it 'ignores unknown attributes' do
```

Comment:
@joneslee85 can you please get back this test?

We can guard for unknown attributes with:

```ruby
def initialize(attributes = {})
    attributes.each do |k, v|
      setter = "#{ k }="
      public_send(setter, v) if respond_to?(setter)
    end
  end
```